cmake/opentelemetry-proto.cmake

```diff
@@ -37,7 +37,7 @@ add_custom_command(
     ${TRACE_PROTO}
 )
 
-include_directories(SYSTEM "${CMAKE_BINARY_DIR}/generated/third_party/opentelemetry-proto")
+include_directories("${GENERATED_PROTOBUF_PATH}")
```

Comment:
It's not a system and there's already variable defined for that same path above.